### PR TITLE
feat: SchemaExample: allow \r \n \t

### PR DESCRIPTION
**Pull request by mschneider82**
#1140

### CODE REVIEW

File: operation.go [hunk 1]
```diff
@@ -544,7 +555,8 @@ func setSchemaExample(param *spec.Parameter, schemaType string, value string) er
 	if param.Schema == nil {
 		return nil
 	}
-	param.Schema.Example = val
+
+	param.Schema.Example = unescapeExample(val)
```
Comment (ubogdan) [inline operation.go:559]: please move the unescapeExample() code here.

File: operation.go [hunk 2]
```diff
@@ -553,7 +565,7 @@ func setExample(param *spec.Parameter, schemaType string, value string) error {
 	if err != nil {
 		return nil // Don't set a example value if it's not valid
 	}
-	param.Example = val
+	param.Example = unescapeExample(val)
 	return nil
```
Comment (ubogdan) [inline operation.go:569]: Why are u making this change here?

**Review by ubogdan — CHANGES_REQUESTED** [review]
See comments...

File: operation.go (hunk 2 above)
Comment (mschneider82) [reply operation.go:569]: To also Support this the replacing in Example() , thats why i put it into a extra func

Comment (mschneider82) [reply operation.go:569]: any advice? remove or keep it for Example?

Comment (ubogdan) [reply operation.go:569]: Please remove the usage on Example. It may introduce a breaking change not covered by unit tests.

Comment (mschneider82) [reply operation.go]: done

**Review by ubogdan — DISMISSED** [review]
LGTM

**Comment by ubogdan** [comment]
Can you please update the README.md accordingly?

**Review by ubogdan — APPROVED** [review]
LGTM

**Comment by ubogdan** [comment]
@mschneider82 Thanks for your contribution.

**Pull request merged**